src/Resources/config/doctrine/Invoice.orm.xml

```diff
@@ -30,9 +30,16 @@
             </cascade>
         </one-to-many>
 
-        <embedded name="channel" class="Sylius\InvoicingPlugin\Entity\InvoiceChannel" />
+        <many-to-one field="channel" target-entity="Sylius\Component\Core\Model\ChannelInterface">
+            <join-column name="channel_id" referenced-column-name="id" nullable="true" />
```

Comment:
```suggestion
            <join-column name="channel_id" referenced-column-name="id" nullable="false" />
```

Reply:
Or even
```suggestion
            <join-column name="channel_id" />
```
as the rest values are the default ones